### ISSUE TITLE
AndroidX issue

### ISSUE THREAD

**Issue by dragon2934** [tanersener/flutter-ffmpeg#14]
Hi, there: 
when I try to import it into my project, it always has this error:

> Android resource linking failed                                       
  warn: removing resource xxx:plurals/error_over_count without required default value.
  warn: removing resource xx:string/error_over_original_count without required default value.
  warn: removing resource xx:string/error_over_original_size without required default value.
  /Users/xxx/.gradle/caches/transforms-1/files-1.1/appcompat-1.0.0-beta01.aar/a72b39a6c9ce382df7d164bcabc15bdd/res/values-v28/values-v28.xml:9:5-12:13: AAPT: error: resource android:attr/dialogCornerRadius not found.
                                                                        
  /Users/xxx/Desktop/work/xxx/build/app/intermediates/incremental/mergeDebugResources/merged.dir/values-v28/values-v28.xml:11: AAPT: error: resource android:attr/dialogCornerRadius not found.
                                                                        
  /Users/xxx/.gradle/caches/transforms-1/files-1.1/design-26.1.0.aar/9482e8475c07035520c8d8edf16e5ac6/res/values/values.xml:159:5-202: AAPT: error: resource android:attr/fontVariationSettings not found.
                                                                        
  /Users/xxx/.gradle/caches/transforms-1/files-1.1/design-26.1.0.aar/9482e8475c07035520c8d8edf16e5ac6/res/values/values.xml:159:5-202: AAPT: error: resource android:attr/ttcIndex not found.
                                                                        
  error: failed linking references.                                     

*******************************************************************************************
The Gradle failure may have been because of AndroidX incompatibilities in this Flutter app.
See https://goo.gl/CP92wY for more information on the problem and how to fix it.
*******************************************************************************************

that says I have to upgrade my whole flutter project to AndroidX, but right now, I can't , any suggestion.

Thanks

**Comment by tanersener** [comment]
Unfortunately changes regarding `AndroidX` support are not released yet. Can you try with the following dependency?

```
  flutter_ffmpeg:
    git:
      url: git://github.com/tanersener/flutter-ffmpeg.git
      ref: development
```

**Comment by dragon2934** [comment]
No, I mean I don't want AndroidX support, but after I import your package,
it cause the project crashed.

**Comment by tanersener** [comment]
Which version of `flutter_ffmpeg` do you use?

**Comment by tanersener** [comment]
`v0.1.1` does not support AndroidX and is tested with old Android support libraries. But logs posted in your first post are printed only when AndroidX is enabled. Since `flutter_ffmpeg v0.1.1` does not enable AndroidX, there must be another plugin or setting which enables AndroidX. 

Can you build your application when `flutter_ffmpeg` dependency is removed? Additionally what happens when you use the following `flutter_ffmpeg` dependency?

```
  flutter_ffmpeg:
    git:
      url: git://github.com/tanersener/flutter-ffmpeg.git
      ref: development
```

**Comment by dragon2934** [comment]
oh, Sorry, as my project has no time for me. I gave up this plugin right
now, will try when I have chance. thanks for your help

**Comment by lhcdims** [comment]
flutter_ffmpeg:
    git:
      url: git://github.com/tanersener/flutter-ffmpeg.git
      ref: v0.1.1
      path: packages/flutter_ffmpeg_https-gpl

Built release apk failed using above settings in pubspec(Error: AndroidX related), but built succeeded using below settings.

  flutter_ffmpeg:
    git:
      url: git://github.com/tanersener/flutter-ffmpeg.git
      ref: development
      path: packages/flutter_ffmpeg_https-gpl

I use flutter_ffmpeg_https-gpl instead of https since I need to use the libx264 coding.  Correct me if I'm wrong, if I want to deploy the apk in Android sdk version 21+, I need to change flutter_ffmpeg_https-gpl to flutter_ffmpeg_https-gpl_lts, otherwise the release apk can only be installed in Android phones with sdk version 24+.

**Comment by tanersener** [comment]
> I use flutter_ffmpeg_https-gpl instead of https since I need to use the libx264 coding. Correct me if I'm wrong, if I want to deploy the apk in Android sdk version 21+, I need to change flutter_ffmpeg_https-gpl to flutter_ffmpeg_https-gpl_lts, otherwise the release apk can only be installed in Android phones with sdk version 24+.

You're right, you need to use `flutter_ffmpeg_https-gpl_lts` package.

Btw, thanks for testing development branch. It is planned to be released next week.

**Comment by tanersener** [comment]
Fixed in `v0.2.0`.
